sqlite3/assets/sqlite3.h

```diff
@@ -39,6 +39,9 @@ int sqlite3_reset(sqlite3_stmt *pStmt);
 
 int sqlite3_column_count(sqlite3_stmt *pStmt);
 int sqlite3_bind_parameter_count(sqlite3_stmt *pStmt);
+sqlite3_char *sqlite3_bind_parameter_name(sqlite3_stmt*, int);
+int sqlite3_bind_parameter_index(sqlite3_stmt*, sqlite3_char *zName);
+int sqlite3_clear_bindings(sqlite3_stmt*);
```

Comment:
I think we can remove this function, we call `sqlite3_reset` when a statement is ran multiple times.